src/pages/index.js

```diff
@@ -15,28 +15,26 @@ const featureRows = [
       imageUrlDark: 'img/community_dark.svg',
       link: 'docs/chain/collator',
       description: (
-        <>
-          Become a collator in for our Peregrine Testnet and later help
-          decentralizing the KILT Blockchain.
-        </>
+        <>Become a collator in for our Peregrine Testnet and later help decentralizing the KILT Blockchain.</>
```

Comment:
I know you just reformatted this, but the content of this sentence sounds a bit outdated. I guess we want to encourage possible collators to join on spiritnet and help decentralizing the KILT blockchain today and not later.

Reply:
that's right! [84382bc](https://github.com/KILTprotocol/docs/pull/69/commits/84382bc6bf0872dfed55b7eabba5fe23b94cacbc)